cluster/README.md

```diff
@@ -11,3 +11,40 @@ Next steps:
 3. Implement your resource handlers by adding code to provision your resources in your resource handler's methods.
 
 Please don't modify files `model.go` and `main.go`, as they will be automatically overwritten.
+
+## Description
+The cluster resource provides access to your cluster configurations. The resource lets you create, edit and delete clusters. The resource requires your Project ID.
+
+## Attributes
+`ID` : Unique identifier of the cluster.<br>
+`MongoURI` : Base connection string for the cluster.<br>
+`MongoURIUpdated` : Timestamp in ISO 8601 date and time format in UTC when the connection string was last updated. The connection string changes if you update any of the other values.<br>
+`MongoURIWithOptions` : Connection string for connecting to the Atlas cluster. Includes the replicaSet, ssl, and authSource query parameters in the connection string with values appropriate for the cluster.<br>
+`Paused` : Flag that indicates whether the cluster is paused or not.<br>
+`ProviderBackupEnabled` : Applicable only for M10+ clusters. Set to true to enable Atlas Cloud Provider Snapshots backups for the cluster. Set to false to disable Cloud Provider Snapshots backups for the cluster. You cannot enable Cloud Provider Snapshots if you have an existing cluster in the project with continuous backups enabled. Note that you must set this value to true for NVMe clusters. The default value is false.<br>
```

Comment:
This may be my deep lack of CFN knowledge but I'm not sure why ProviderBackupEnabled would be in attributes and not a Parameter?     The rest of the attributes seem to be information returned, while ProviderBackupEnabled is set by the user, hence a parameter?

Reply:
It's true, it's an optional parameter, gotta change it, thanks